docs/spfx/publish-to-marketplace-faq.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Publish SharePoint Framework applications to the marketplace - FAQ
+description: Frequently asked questions about publishing SharePoint Framework applications to the marketplace
+ms.date: 06/25/2020
+ms.prod: sharepoint
+localization_priority: Priority
+---
+
+# Frequently asked questions
+
+## Is it required to configure web parts to be isolated for publishing in the marketplace
```

Comment:
Issue: don't start a doc with two headings.

- Docs must start with an H1 that matches the doc title
- Docs should have an overview statement between the title & the first subheading